custom_components/elegoo_printer/image.py

```
@@ -0,0 +1,143 @@
+            return image.content
+        return None
+
+    @property
+    async def async_image(self) -> bytes | None:
+        """Return image."""
+        return await self.get_image()
+
+    # @property
+    # def image_last_updated(self) -> datetime | None:
+    #     """The time when the image was last updated."""
+    #     return self.coordinator.get_model().cover_image.get_last_update_time()
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)